### DISCUSSION TITLE
菜单里没有“发送到aria”

### DISCUSSION THREAD

**Discussion by ttttmr** [AlistGo/alist#4864] [Q&A] [answered]
![image](https://github.com/alist-org/alist/assets/32825326/977c8cde-4311-4e29-9dbb-87b57d6248d5)

本地设置和后台都配置了aria，但没有发送的菜单，是要在哪开启吗

用的docker版本v3.23.0

**Comment by anwen-anyi** [comment, marked as answer]
在右下角点击 `开关复选框 ` --> `选择列表中的资源` -->  底部会出现一排列表选择里面的`下载`有一个发送到Aria2
